tests/test_init.py

```
@@ -5497,9 +6370,17 @@ class DeviceRegistry:
         def __init__(self) -> None:
             """Provide a fake device registry object for the collision test."""
 
-        def async_update_device(self, *a, **k) -> Never:
+        def async_update_device(self, *a: object, **k: object) -> Never:
```

Ruff (0.16.1)

[warning] Unused method argument: `a`

(ARG002)